cypress/support/settings.ts

```diff
@@ -111,8 +111,8 @@ Cypress.Commands.add("setSettingValue", (
     level: SettingLevel,
     value: any,
 ): Chainable<void> => {
-    return cy.getSettingsStore().then(async (store: typeof SettingsStore) => {
-        return store.setValue(name, roomId, level, value);
+    return cy.getSettingsStore().then((store: typeof SettingsStore) => {
+        return cy.wrap(store.setValue(name, roomId, level, value));
```

Comment:
Works on my machine™

Probably when entering the `Chainable` world of Cypress everything has to be `Chainable`?